encoders/image/FaceNetEncoder/__init__.py

```diff
@@ -0,0 +1,62 @@
+__copyright__ = "Copyright (c) 2021 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import numpy as np
+import torch
+
+from jina.executors.decorators import batching, as_ndarray
+from jina.executors.encoders.frameworks import BaseTorchEncoder
+from jina.executors.devices import TorchDevice
+
+
+class FaceNetEncoder(BaseTorchEncoder, TorchDevice):
+    """FaceNetEncoder encodes face images into an embedding with 512 dimensions.
+    Works best with outputs from FaceNetSegmenter (e.g. images with faces, 160x160, normalized).
+
+    - Input shape: `BatchSize x (Channels x Height x Width)`
+    - Output shape: `BatchSize x 512`
+
+    `Channels` dimension can be changed (e.g. set `channel_axis` to 1 for channels-first mode instead of channels-last).
+
+    :param pretrained_weights: Weights to use for face embedder. Options: "vggface2", "casia-webface"
+    :param channel_axis: Axis of channels in the image. Default is 3 (channels-last), use 1 for channels-first.
+    """
+
+    def __init__(self,
+                 pretrained_weights: str = 'vggface2',
+                 channel_axis: int = 1,
+                 *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.pretrained_weights = pretrained_weights
+        self.channel_axis = channel_axis
+
+        self.embedder = None
```

Comment:
Claiming the `embedder` in `__init__` will lead to storing it when dumping. This is not what we want because the model can always be loaded on the spot. BTW, what about using `model` instead of `embedder`?
```suggestion
```

Reply:
Sure, I wasn't familiar with serialization logic. pickle would serialize this property in `BaseExecutor.save` if it is defined in `__init__`.